### ISSUE TITLE
Include underscore not working

### ISSUE THREAD

**Issue by harlancleiton** [rhwilr/adonis-bumblebee#67]
First of all, i'm not a native speaker and i used google translator to help me write this issue.
By now congratulations on the package. It is very useful and practical

I am having difficulties adding an underscore property in the available include

On my Transformer:
```
static get availableInclude () {
     return ['amount_used', 'administrator']
   }

includeAmountUsed (coupon) {
     const {amount_used} = coupon.toJSON ()

     return amount_used
   }
```

In Controller:
```
async show ({...}) {
...
const transformed = await transform
         .include ('amount_used, administrator')
         .item (coupon, CouponTransformer)

       return response.json (transformed)
}
```

JSON response example:
```
{
  "administrator": {
    "id": 12,
    "first_name": "Kate",
    "last_name": "Julien",
    "email": "mu@sun.be",
    "cpf": "58278510784",
    "date_birth": "2102-05-05T03: 00: 00.000Z"
  },
  "id": 8,
  "code": "jTh) vM5!",
  "valid_from": null,
  "valid_until": null,
  "type": "currency",
  "quantity": 995,
  "discount": "78.53"
}
```

Already tried a debug or add a console log in includeAmountUsed, but the function is not called. I have also tried renaming the function to includeAmount_used, includeAmount_Used. 
If rename the property to amountUsed works perfectly, but I wanted JSON to be amount_used, can you help me? Thanks

**Comment by rhwilr** [comment]
Hi @harlancleiton 

This is currently not possible, and I don't think we can easily support this without braking existing functionality. 

It might be possible to implement this with a custom [Serializer](https://github.com/rhwilr/adonis-bumblebee#serializers).
You should be able to write a serializer that transforms all properties to snake_case before returning the response.

However, I do think we should support includes with underscores. I might take a look at it. I'd also happily accept a PR.

**Comment by harlancleiton** [comment]
@rhwilr 

I understand, I thought this note at [README](https://github.com/rhwilr/adonis-bumblebee#including-data) described this functionality

> Note: If you have hyphen or underscore separated properties, you would name the include function in camelCase. The conversion is done automatically.

**Comment by rhwilr** [comment]
I published version [2.1.0](https://github.com/rhwilr/adonis-bumblebee/releases/tag/2.1.0) that now supports snake_case include properties.

It should now work as you'd expect and your example from the initial post should work:

```js
static get availableInclude () {
    return ['amount_used', 'administrator']
}

includeAmountUsed (coupon) {
    const {amount_used} = coupon.toJSON ()

    return amount_used
}
```
